scripts/coco

```diff
@@ -214,11 +214,12 @@ if hasattr(parsed_args, "func"):
                     parsed_args.style, STYLES
                 )
             )
+            sys.exit(1)
     if not parsed_args.silent:
         print("Done.")
 else:
     parser.print_help()
     print("\ncoco: error: an endpoint must be specified")
-    exit(1)
+    sys.exit(1)
 
-exit(0)
+sys.exit(not success)
```

Comment:
where is `not success` defined?

Reply:
`not` is a logical negation, `success` is defined above

Reply:
Wow, syntax highlighting is my friend.

Thank you!